exporters/otlpgrpcexporter/config.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package otlpexporter // import "go.opentelemetry.io/collector/exporter/otlpexporter"
+package otlpgrpcexporter // import "go.opentelemetry.io/collector/exporters/otlpgrpcexporter"
```

Comment:
Do we rename the package only but keep `otlp` as the component name?

Reply:
`otlp` in the config applies to both grpc and http exporters, my opinion is that it's fine to separate config from package name, but i don't feel strongly about it. i can revert if that's preferable.

Reply:
> otlp in the config applies to both grpc and http exporters, 

otlphttpexporter uses `otlphttp` https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/otlphttpexporter/factory.go#L33

I think we should keep module names consistent with the collector component names. I would vote for keeping `otlpexporter` without renaming it. This seems like the most important one

Reply:
sorry i was thinking of the receiver 😅

Reply:
I think we should keep it as `otlpexporter`. It would otherwise be the only exporter with a mismatch between the Go module name and the configuration identifier.